docker/data-science/notebooks/solr.py

```diff
@@ -344,12 +332,12 @@ def spell_check(self, collection, query):
                 for r in response["spellcheck"]["collations"]
                 if r != "collation"}
 
+    def generate_query_time_boost(query):
+        return f"sum(1,query({query})"
+    
     def print_status(self, solr_response):
         print("Status: Success" if solr_response["responseHeader"]["status"] == 0 else "Status: Failure; Response:[ " + str(solr_response) + " ]" )
 
-    def docs_from_response(self, response):
-        return response["response"]["docs"]
-    
     def tag_query(self, collection_name, query):
         url_params = "json.nl=map&sort=popularity%20desc&matchText=true&echoParams=all&fl=id,type,canonical_form,surface_form,name,country:countrycode_s,admin_area:admin_code_1_s,popularity,*_p,semantic_function"
```

Comment:
todo: verify. I thought we moved all of these into the tag handler. If so, these params aren't needed if they are the hard-coded defaults.

Reply:
This orphaned function has no been deleted